### ISSUE TITLE
No "Main landmark" is defined for the "Browse" workshop page: A11y_Maow_Browse Workshop_Bypass blocks. 

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/moaw#115]
### GitHub Tags
#A11yTCS; #A11ySev2; #ChromiumEdge; #WCAG2.4.1; #Win11; #GH_Maow_Web_Mar24; #Maow; #Bypass Blocks; #Benchmark; #AI Limited; #DesktopWeb; #A11yMAS;

### Environment Details:
Application Name: Maow
#URL: https://moaw.dev/catalog/
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)
Microsoft Edge: Version 122.0.2365.92 (Official build) (64-bit)

### Repro Steps:
1. Hit the URL https://moaw.dev/
2. Moaw screen will get open.
3. TAB till "Browse workshops" control and press ENTER.
4. TAB till "Header" section.
5. Verify that the "Main landmark" is defined for the "Browse" workshop page or not.
### Actual Result:
No "Main landmark" is defined for the "Maow" page.
### Expected Result:
"Main landmark" should be defined for "Browse" workshop page:
### User Impact:
Screen reader users will not able to navigate the main information easily if the main landmark is not defined for the page.

### Attachment:
![Bug_115](https://github.com/microsoft/moaw/assets/124244478/0446a5fe-a8ab-45d5-900c-a30996ebce97)

**Comment by sinedied** [comment]
@AnKushSingh05 this one has been fixed, could you validate it?

**Comment by AnKushSingh05** [comment]
Verified the issue in below environment
https://moaw.dev/
Issue is fixed. Hence closing the bug

![Screenshot (1958)](https://github.com/microsoft/moaw/assets/124244478/165b7d05-ec58-4190-8d81-33899ee15ac1)
